hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerStateMachine.java

```diff
@@ -689,6 +698,8 @@ private synchronized void computeAndUpdateLastAppliedIndex(
         }
       }
     }
+    this.metrics.updateApplyTransactionMapSize(applyTransactionMap.size());
+    this.metrics.updateRatisTransactionMapSize(ratisTransactionMap.size());
```

Comment:
With new implementation to fix issue, this may not be required. But still if any blocking due to applyTransaction pending size grows, this also may increase.
IMO, this might not be required now.
A metrics for time taken by applyTransaction is already captured by org.apache.hadoop.ozone.protocolPB.OzoneManagerRequestHandler#handleWriteRequest --> getValidateAndUpdateCacneLatencyNs()

Reply:
I would prefer keep the metrics.